src/platform/unittest/TlsTest.cpp

```diff
@@ -2048,7 +2048,7 @@ ValidateSecConfigStatusSchannel(
 TEST_F(TlsTest, PlatformSpecificFlagsSchannel)
 {
     for (auto TestFlag : { QUIC_CREDENTIAL_FLAG_ENABLE_OCSP, QUIC_CREDENTIAL_FLAG_USE_SUPPLIED_CREDENTIALS,
-        QUIC_CREDENTIAL_FLAG_USE_SYSTEM_MAPPER,
+        QUIC_CREDENTIAL_FLAG_USE_SYSTEM_MAPPER, QUIC_CREDENTIAL_FLAG_SCHANNEL_SERIALIZED_PEER_CERT,
```

Comment:
We need more tests than this. We need to try calling a connection with the new flag. I expect that test to pass on WS2022, but fail on WS2019.

Reply:
Added a Tls unit test which gets to the heart of the feature, and the existing unit tests caught regressions.